common/unit-test/softwarecontainer-common_unittest.cpp

```diff
@@ -57,3 +83,81 @@ TEST_F(SoftwareContainerCommonTest, ReturnCodeisSuccess) {
     ASSERT_TRUE(isSuccess(ReturnCode::FAILURE) == false);
 }
 
+/*
+ * Test that a directory is only seen as a directory and not as a file, socket etc.
+ */
+TEST_F(SoftwareContainerCommonTest, DirIsOnlyDir) {
+    std::string dirStr = getTempPath(true, false);
```

Comment:
maybe using alias for readability would be useful 
`isDirectory = true`
`shouldUnlink = dalse`
`getTempPath(isDirectory, shouldUnlink);`